src/components/common/EmailPrompt.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <Dialog v-model:visible="visible" :closable="false" :header=title :style="{ width: '25rem' }">
+        <span class="p-text-secondary block mb-5">{{ dialog }}</span>
+        <div class="flex align-items-center gap-3 mb-3">
+            <label for="email" class="font-semibold w-6rem">Email</label>
+            <InputText v-model:model-value="email" id="email" class="flex-auto" />
+        </div>
+        <div><span v-if="emailValidationError" class="mave-field-error">{{ emailValidationError }}</span></div>
+        <div class="flex justify-content-end gap-2">
+            <Button type="button" label="Ignore" severity="secondary" @click="ignoreEmail"></Button>
```

Comment:
```suggestion
            <Button type="button" label="Not now" severity="secondary" @click="ignoreEmail"></Button>
```

Reply:
not super sure on the wording here. any brilliant ideas?

Reply:
Not really, the only other thing I can think of is something like `Maybe later`.